components/calendar/calendar.js

```diff
@@ -160,6 +161,11 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			 * @type {string}
 			 */
 			label: { attribute: 'label', reflect: true, type: String },
+			/**
+			 * Initial date to show. If provided, overrides logic for determining the default year and month to show.
+			 * @type {string}
+			 */
+			initialValue: { attribute: 'initial-value', type: String },
```

Comment:
Could we just use `selectedValue` for this, or is there a case where `selectedValue` and `initialValue` would be different?

Reply:
I think they are slightly different. If we use `selectedValue`, then a date would actually be selected when the user is switching weeks or months in the Calendar tool. A subtle behaviour change. It might be ok, but I am also a _little_ worried about other things that could change as a result of that.